docker-action.yml.mustache

```diff
@@ -1,4 +1,4 @@
-name: CI
+name: Docker-based CI
```

Comment:
I am not sure it is a good idea to change this identifier: this is just the internal name of the workflow within the Docker Action interface, and changing this would broke the badge URLs:

https://github.com/coq-community/templates/blob/87ce3a2e7127dee0770dd2901341f0102edc37aa/README.md.mustache#L29-L30

Reply:
and moreover, I'm not sure if a space would be allowed :-/

Reply:
Oh, I forgot about the badge!

But no, this is not a purely internal name and spaces are allowed. See the test PR:

![2020-12-08-110401_702x456_scrot](https://user-images.githubusercontent.com/1108325/101469530-27e86380-3945-11eb-9a01-57e162e52ca9.png)

Reply:
In the future, a Coq project may have 5+ different Action workflows, possibly driven mostly by our templates. This is why I think it's important to have a descriptive name for each workflow - inexperienced Coq project maintainers are sure to be overwhelmed otherwise.